pymem/__init__.py

```diff
@@ -78,6 +78,25 @@ def list_modules(self):
         modules = pymem.process.enum_process_module(self.process_handle)
         return modules
 
+    def resolve_offsets(self, base_offset, offsets, *, is_64_bit = True):
+        """Resolves a list of pointers; commonly one from cheat engine
+
+        Args:
+            base_offset (int): The base address offset
+            offsets (list[int]): List of offsets
+            is_64_bit (bool, optional): If the process is 64_bit or not. Defaults to True.
+        """
+        if is_64_bit:
+            read_method = self.read_longlong
+        else:
+            read_method = self.read_int
```

Comment:
You probably want to read unsigned types since the address cannot be negative.

Also, it might be interesting to see how it's done here: https://gitlab.com/v01d-gm/trainer-base/-/blob/main/trainerbase/memory.py#L26. Tested on a significant number of "modifications" for games.